tests/helpers/sharingHelper.js

```diff
@@ -130,11 +130,23 @@ const getShareIdToken = (resource, resourceType) => {
   return { shareId, shareToken }
 }
 
+/**
+ *
+ *
+ * @returns {String} expiration data (10+ days from current Date)
+ */
+const getExpirationDate = () => {
+  const currentDate = new Date()
+  currentDate.setDate(currentDate.getDate() + 10)
+  return currentDate.toISOString().slice(0, 10)
```

Comment:
I suppose the date is something like `2022-10-20` which happens to be a 10-char-long string.
Future readers (and me) are going to wonder which "10" (or "10s") to change if the date offset needs to be different in the future. A comment will help to explain that `slice(0, 10)` is nothing to do with the `+ 10` in the line above.

Reply:
@phil-davis sure thanks for reporting !!